rita/src/rita_client/exit_manager/mod.rs

```diff
@@ -1,16 +1,16 @@
 use actix::prelude::*;
+use actix::registry::SystemService;
 
-use reqwest;
-
-use althea_types::{
-    ExitClientDetails, ExitClientIdentity, ExitDetails, ExitServerReply, ExitState,
-};
+use althea_types::{ExitClientDetails, ExitClientIdentity, ExitState};
 
 use settings::{RitaClientSettings, RitaCommonSettings};
 use SETTING;
 
 use rita_client::rita_loop::Tick;
 use rita_client::traffic_watcher::{TrafficWatcher, Watch};
+use rita_common::http_client::{ExitSetupRequest, GetExitInfo, HTTPClient};
+
+use futures::Future;
```

Comment:
so these are native futures as opposed to actix futures now?

Reply:
they are native futures as opposed to doing blocking operations within actors